packages/backend-common/api-report.md

```diff
@@ -342,6 +342,8 @@ export class Git {
     force?: boolean;
   }): Promise<void>;
   // (undocumented)
+  branch(options: { dir: string; ref: string }): Promise<void>;
```

Comment:
API report generated with `yarn build:api-reports packages/backend-common` since doing a full run crashed Node with an `FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory` :/ I can only assume that only covering backend-common is sufficient?

Reply:
Impressive as in not being able to do a full API report or that it was possible for me to do backend-common? :) Anyhow I'm running everything on a Thinkpad T480 with 16G of RAM. Node is 16.14.2.

Reply:
Impressive that it's not enough memory to to do a full report :joy: I wonder if you might need to set some sane limits for the node VM but it's odd. Think this is fine anyways.